src/js/product.js

```diff
@@ -1,8 +1,26 @@
 import { setLocalStorage } from "./utils.mjs";
+import { getLocalStorage } from "./utils.mjs";
 import { findProductById } from "./productData.mjs";
 
```

Comment:
Good job in adding this function. You helped me realize that it is better to check whether the cart is empty or not by reviewing the data in the localStorage